packages/cron/src/lib/email.js

```diff
@@ -0,0 +1,89 @@
+import debug from 'debug'
+
+export const emailType = {
```

Comment:
Given this represents an postgres type it should be part of the DB package and imported by client packages.
What you think?

Reply:
Agree. Have moved to db client types.